src/components/widget.js

```diff
@@ -447,7 +445,7 @@ class Widget extends LitElement {
       platform: this.platform,
     });
 
-    if (status === 401) {
+    if (status === 401 || !response?.workspace) {
```

Comment:
Given the changes [here](https://github.com/orbit-love/orbit-browser-extension/pull/63/commits/f4a61ae2a048655b4d951817f219989d4abe5411#diff-16f022ec89282b37542e3374910a5e4a4cf37596abf8f723f0d7f8ab1096f853L193-R193), is that still necessary?

Reply:
Exactly that, yep ^^ You nailed it. But will add as a comment for posterity